packages/flutter_tools/test/general.shard/web/chrome_test.dart

```diff
@@ -61,29 +62,50 @@ void main() {
     resetChromeForTesting();
   });
 
-  test('can launch chrome and connect to the devtools', () async {
-    processManager.addCommand(const FakeCommand(
+  Future<Chrome> _testLaunchChrome(String userDataDir) {
```

Comment:
I would just leave this as public and hoist to the top level scope at the bottom of the file. It should not capture any of the locally defined values.

Reply:
Since this is a convenience function, being able to access `processManager` and `chromeLauncher` is part of its value. It would be less convenient if these things had to be passed at each call site. Is there danger with capturing them?

Reply:
BTW "capture" is probably not the right word. They are _in scope_. They are not captured in the closure sense.

Reply:
Done though